lonestar/analytics/cpu/matrix-completion/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(matrixcompletion-sgd-cpu matrix_completion_sgd_cli.cpp)
+add_dependencies(apps matrixcompletion-sgd-cpu)
+target_link_libraries(matrixcompletion-sgd-cpu PRIVATE Katana::galois lonestar)
+
+add_test_scale(small1 matrixcompletion-sgd-cpu INPUT Epinions_dataset INPUT_URI "${BASEINPUT}/propertygraphs/Epinions" --edgePropertyName=value --algo=sgdByItems)
```

Comment:
I don't think `propertygraphs/Epinions` is in the inputs archive. It will need to be updated.

Reply:
I added Epinions and uploaded a new input collection to the input archive.